iceoryx_hoofs/test/moduletests/test_posix_posix_call.cpp

```diff
@@ -473,3 +479,39 @@ TEST_F(PosixCall_test, CallingFunctionWithMultipleFailureReturnValuesWhereFailur
 
     EXPECT_TRUE(internal::GetCapturedStderr().empty());
 }
+
+TEST_F(PosixCall_test, ErrnoIsSetFromReturnValueWhenFunctionHandlesErrnosInReturnValue_GoodCase)
+{
+    internal::CaptureStderr();
+
+    constexpr int RETURN_VALUE = 41;
+
+    iox::posix::posixCall(returnValueIsErrno)(RETURN_VALUE)
+        .failureReturnValue(RETURN_VALUE - 1)
```

Comment:
Just a question, should this also be done for `successReturnValue`?

Reply:
Yes of course, thanks!